utils/build-apple-framework.sh

```diff
@@ -46,7 +46,10 @@ function build_host_hermesc {
 # Utility function to configure an Apple framework
 function configure_apple_framework {
   local build_cli_tools enable_bitcode
-  if [[ $1 == iphoneos ]]; then
+  local catalyst="false"
+  local platform=$1
+
+  if [[ $1 == iphoneos || $1 == catalyst ]]; then
```

Comment:
Perhaps also replace further use of `$1` with the `$platform` alias here?

Reply:
I would actually not use `$platform` here. Further explanation on line 62.